### PR TITLE
Kinetic Surface Reconstruction Plugin

### PR DESCRIPTION
**Pull request by soesau**
## Summary of Changes

Copied from siggraph24 branch
added parameter estimation

## Release Management

* Affected package(s): Lab, Kinetic Space Partition, Kinetic Surface Reconstruction, Point Set 3

### CODE REVIEW

**Comment by sloriot** [comment]
Tests failing in CGAL-6.0-Ic-323

See [1](https://cgal.geometryfactory.com/CGAL/testsuite/summary-6.0-Ic-323.html?package=Kinetic_surface_reconstruction_Examples) and [2](https://cgal.geometryfactory.com/CGAL/testsuite/summary-6.0-Ic-323.html?package=Shape_detection) for example.

File: Shape_detection/include/CGAL/Shape_detection/Region_growing/Point_set/Least_squares_plane_fit_sorting.h [hunk 1]
```diff
@@ -174,28 +185,76 @@ namespace Point_set {
     const Seed_range &ordered() {
       return m_ordered;
     }
+
+    /*!
+      \brief the average of the maximal point to fitted plane distance in each neighborhood.
+    */
+
+    const double mean_distance() {
```
Comment (sloriot) [inline Shape_detection/include/CGAL/Shape_detection/Region_growing/Point_set/Least_squares_plane_fit_sorting.h:193]:
```suggestion
    double mean_distance() {
```

File: Shape_detection/include/CGAL/Shape_detection/Region_growing/Point_set/Least_squares_plane_fit_sorting.h [hunk 2]
```diff
@@ -174,28 +185,76 @@ namespace Point_set {
...
+      return mean_d;
+    }
+
+    /*!
+      \brief the average of the maximal normal deviation to fitted plane in each neighborhood.
+    */
+    const double mean_deviation() {
```
Comment (sloriot) [inline Shape_detection/include/CGAL/Shape_detection/Region_growing/Point_set/Least_squares_plane_fit_sorting.h:200]:
```suggestion
    double mean_deviation() {
```

**Comment by afabri** [comment]
Warnings in[ I-c-337](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-6.0.1-Ic-337/Lab_Demo/TestReport_lrineau_ArchLinux-clang-CXX20-Release.gz)

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0.1-Ic-345

**Pull request merged**